client/src/locales/en-US.json

```diff
@@ -126,7 +130,7 @@
             "newVersionAvailable": "New update available"
         },
         "organizationList": {
-            "title": "Your organizations",
+            "title": "Access to your organizations",
```

Comment:
Depending on the wanted formulation, change this or the french version

`Access your organizations` + `Accédez à vos organisations`
or
`Access to your organizations` + `Accès à vos organisations`